handsontable.d.ts

```diff
@@ -83,7 +84,7 @@ declare namespace _Handsontable {
     toVisualRow(row: number): number;
     unlisten(): void;
     updateSettings(settings: object, init: boolean): void;
-    validateCells(callback: () => void): void;
+    validateCells(callback: (valid: boolean) => void): void;
```

Comment:
Line 339 in the tests hasn't yet been updated to reflect this change in signature.